examples/run_v2_5_0_all_sets_E3SM_machines.py

```diff
@@ -0,0 +1,236 @@
+import os
+
+from acme_diags.parameter.annual_cycle_zonal_mean_parameter import ACzonalmeanParameter
```

Comment:
I agree we should use `acme_diags` since that's what is needed for v2.5.0. However, the `complete_run` test, when run in the latest development environment, expects `e3sm_diags`. Maybe the best solution is to have `complete_run` use a copy of this file that trades `acme_diags` for `e3sm_diags`?

Reply:
yeah, there is some inconvenience during transition from importing `acme_diags` to `e3sm_diags`. The run script is fur v2_5_0, so i think it is fine to keep `import acme_diags`. I'm not sure what does it mean by "use a copy of this file that trades acme_diags for e3sm_diags"...

Reply:
@chengzhuzhang I meant meant an exact copy of this file named something like `examples/run_v2_5_0_all_sets_E3SM_machines_updated_imports.py`, and then replace `acme_diags` calls with `e3sm_diags`. Then, in `tests/complete_run.py` use `from examples.run_v2_5_0_all_sets_E3SM_machines_updated_imports import run_lcrc`

I could do that in a follow-up PR, and update the test's expected images too.

Reply:
Good reminder! the expected images need to be replaced as well. I think it makes sense to use this branch/PR to include the proposed change? Would you also help address the review comments? They are all good!

Reply:
@chengzhuzhang yes I will push a commit addressing my suggestions.

Reply:
@chengzhuzhang To update the tests, I need write permissions to `/lcrc/group/e3sm/public_html/e3sm_diags_test_data/unit_test_complete_run/expected/all_sets`